CHANGELOG.md

```diff
@@ -13,6 +13,24 @@ next release
 
 ### UI Changes
 
+1.40.0 (2022-12-07)
+-------------------
+
+#### New Features
+
+* Release signing ([@jkowall](https://github.com/jkowall) in [#4033](https://github.com/jaegertracing/jaeger/pull/4033))
+
+#### Bug fixes, Minor Improvements
+
+* Fix cassandra schema scripts to be able to run from a remote node ([@yurishkuro](https://github.com/yurishkuro) in [#4087](https://github.com/jaegertracing/jaeger/pull/4087))
+* Catch panic from Prometheus client on invalid label strings ([@yurishkuro](https://github.com/yurishkuro) in [#4051](https://github.com/jaegertracing/jaeger/pull/4051))
+* [Bug]: span tags of type int64 may lose precision #3958 PR No 2 ([@shubbham1215](https://github.com/shubbham1215) in [#4034](https://github.com/jaegertracing/jaeger/pull/4034))
```

Comment:
I suggest removing the "#3958 PR No 2". I'm also not sure if we need "[Bug]: ".

Reply:
agree. thanks for the suggestion